frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -630,68 +609,62 @@ class DashboardHeaderContainer extends Component<DashboardHeaderProps> {
       );
     }
 
-    return buttons;
-  }
+    return { buttons };
+  };
 
-  saveAsPDF = async () => {
-    const { dashboard } = this.props;
-    const cardNodeSelector = `#${DASHBOARD_PDF_EXPORT_ROOT_ID}`;
+  const saveAsPDF = async () => {
+    const cardNodeSelector = "#Dashboard-Cards-Container";
     await saveDashboardPdf(cardNodeSelector, dashboard.name).then(() => {
       trackExportDashboardToPDF(dashboard.id);
     });
   };
```

Comment:
nit: think it'd be way more discoverable if we keep all callbacks next to each other